duckduckgo/iOS
# Sync UI PDF Generation

Task/Issue URL: https://app.asana.com/0/414235014887631/1204117883679510/f
Tech Design URL:
CC:

**Description**:

Adds PDF Generation and updates save recovery key screen.

**Steps to test this PR**:
1. Smoke test all of Sync UI
2. After turning on sync or connecting a device, save the PDF and check the contents
3. Make sure copy button works



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
